app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -474,15 +462,22 @@ private void takeScreenshotOfErrorPages(UiDevice device) throws UiObjectNotFound
 
             TestHelper.inlineAutocompleteEditText.setText("error:"+ error.value);
             device.pressKeyCode(KEYCODE_ENTER);
-
-            onWebView()
-                    .withElement(findElement(Locator.ID, "errorTitle"))
-                    .perform(webClick());
-
-            onWebView()
-                    .withElement(findElement(Locator.ID, "errorTryAgain"))
-                    .perform(webScrollIntoView());
-
+            TestHelper.webView.waitForExists(waitingTime);
+
+            if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
+                UiObject tryAgainBtn = device.findObject(new UiSelector()
+                        .resourceId("errorTryAgain")
+                        .clickable(true));
+                tryAgainBtn.waitForExists(waitingTime);
+            } else {
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 13 should be on the same line as the next part of a multi\-block statement\.